### PR TITLE
Create record position field

### PR DESCRIPTION
**Pull request by thomtrp**
- on non system standard objects 
- on custom objects

Tested with:
- custom object creation
- seeds with standard objects

### CODE REVIEW

File: packages/twenty-server/src/metadata/object-metadata/object-metadata.service.ts [hunk 1]
```diff
@@ -291,6 +291,22 @@ export class ObjectMetadataService extends TypeOrmQueryService<ObjectMetadataEnt
             workspaceId: objectMetadataInput.workspaceId,
             defaultValue: { type: 'now' },
           },
+          {
```
Comment (Weiko) [inline packages/twenty-server/src/metadata/object-metadata/object-metadata.service.ts:294]: You'll also need to generate the migration (see below)

**Review by Weiko — COMMENTED** [review]
Overall lgtm but do we want to use feature flags for this?

**Comment by thomtrp** [comment]
> Overall lgtm but do we want to use feature flags for this?

@Weiko not sure we need it. This is a new field that we can choose to use or not directly in frontend?

**Review by charlesBochet — APPROVED** [review]
Looks great!

**Pull request merged**